ETL-Project/src/pipeline/extract_from_db.py

```diff
@@ -0,0 +1,46 @@
+
+import sys
+path = 'C:\\Users\\bikas\\Downloads\\Data_Enginner-Internship\\DBMS\\week3\\OLAP Design\\Assignmanets_all\\ETL-Project\\'
+sys.path.append(path)
+from src.util.connector import * 
+
+
+
+def extract_data_db():
+    source_conn = connect1()
+    dest_conn = connect()
```

Comment:
How about we make a single connect function that takes db secrets and returns the connection?